docs/preview/index.md

```diff
@@ -147,6 +147,9 @@ firefox https://127.0.0.1
 - You may see the following error message `TypeError: Object.fromEntries is not a function`.
   This means you need to update your base Invenio docker image because node 13+ is needed.
   Run `docker pull inveniosoftware/centos7-python:3.6` or `docker pull inveniosoftware/centos8-python:3.7` (or python 3.8)  before running `invenio-cli containerize --pre` again.
+- You may see `SystemError: Parent module 'setuptools' not loaded, cannot perform relative import`
+  at the dependency locking step when running `invenio-cli containerize --pre`. This depends on your version of `setuptools` (bleeding edge causes this)
```

Comment:
I think this was happening both in `containerize` and `local`...

Reply:
:+1: added to the other one too